arch/risc-v/src/common/espressif/esp_twai.c

```diff
@@ -252,6 +252,7 @@ static void esp_twai_reset(struct can_dev_s *dev)
 
   ret = twai_hal_init(&priv->ctx, &hal_config);
   assert(ret);
```

Comment:
```suggestion
  ASSERT(ret);
```
and revert the change at line 255